### PR TITLE
Movie description page structure&styles and functional added

### PR DESCRIPTION
**Pull request by Oleh020**

### CODE REVIEW

File: moviePage.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="icon" type="image/png" sizes="32x32" href="./icons/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./icons/favicon/favicon-16x16.png">
+    <link rel="icon" type="icon" href="./icons/favicon/favicon.ico">
+    <title> </title>
+</head>
+<body>
+    <Section id="movie__info">
+    </Section>
+    <script async src="./js/moviePage.js" type="module"></script>
+    <script async src="./js/constants.js" type="module"></script>
```
Comment (carbonid1) [inline moviePage.html]: I'm curious why the constants are loaded only the second

File: styles/moviePage.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+* {
+    box-sizing: border-box;
+    margin: 0 auto;
+}
```
Comment (carbonid1) [inline styles/moviePage.css]: this piece should be in global styles

File: moviePage.html (hunk 1 above)
Comment (Oleh020) [reply moviePage.html]: fixed

File: styles/moviePage.css (hunk 2 above)
Comment (Oleh020) [reply styles/moviePage.css]: fixed

**Review by carbonid1 — APPROVED** [review]
awesome!

**Pull request merged**